jplag/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -158,7 +158,7 @@ private Optional<Submission> loadBaseCode(Set<File> submissionDirectories, Set<F
                 logger.info("Submission \"{}\" is the specified basecode, it will be skipped during comparison.", removed.getName());
             }
         }
-        return baseCodeSubmission;
+        return Optional.of(baseCode);
```

Comment:
ofNullable here?

Reply:
Thanks for spotting that. Had some other refactoring before where `ofNullable` was not necessary anymore.